pop_tools/modify_pop_for_xgcm.py

```diff
@@ -0,0 +1,98 @@
+"""
+Module for modifying pop model output to be compatible with xgcm
```

Comment:
I don't think this module is named appropriately. How about `xgcm_util.py`?

Reply:
> How about xgcm_util.py?

👍